scout/constants/query_terms.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+
+# These terms are used by the function that creates a variant query dictionary in adapter
+
+SECONDARY_CRITERIA = [
```

Comment:
:+1: Perhaps also a 0-level for gene, genlist etc for clarity?

Reply:
E.g. one could argue that inheritance pattern might often be used as a 0th level excluding criteria. Users are typically surprised to find variants not following inheritance pattern after hitting this, and it is not used in the clinical filter. Im not saying we should change it now, but perhaps convenient to have it explicit!

Reply:
I think we have to discuss in person about this because I don't understand. Same for the soft, hard excluding criteria. : -/